include/opw_kinematics/opw_parameters_examples.h

```diff
@@ -24,8 +33,7 @@ Parameters<T> makeIrb2400_10()
 }
 
 template <typename T>
-__attribute__((deprecated("UN-TESTED")))
-Parameters<T> makeFanucR2000iB_200R()
+DEPRECATED Parameters<T> makeFanucR2000iB_200R()
```

Comment:
It seems `DEPRECATED` does not support specifying a message.

Should not be hard to add (just do something like `DEPRECATED(x)` and pass `x` on if the target platform supports it.

Reply:
I updated deprecated to take a message.